### PR TITLE
Alembic invocations

### PR DESCRIPTION
**Pull request by TheSuperiorStanislav**

### CODE REVIEW

File: saritasa_invocations/alembic.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                if os.path.isfile(f"{migrations_folder}/{name}")
+            ],
+        )
+        + 1,
+    )
+    rev_id = rev_id.rjust(4, "0")
+    command = (
+        f'revision --autogenerate --message "{message}" --rev-id={rev_id}'
+    )
+    run(context, command=command)
```
Comment (ron8mcr) [inline saritasa_invocations/alembic.py:60]: Let's add further instructions for developer like:
* Verify migrations
* Remove alembic's messages

File: saritasa_invocations/alembic.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            "# ### commands auto generated by Alembic - please adjust! ###",
+            "# ### end Alembic commands ###",
+        ),
+    )
+    files_to_clean = collections.defaultdict(set)
+    for filename in os.listdir(migrations_folder):
+        file_path = f"{migrations_folder}/{filename}"
+        if not os.path.isfile(file_path):
+            continue
+        printing.print_success(f"Checking {file_path}")
```
Comment (ron8mcr) [inline saritasa_invocations/alembic.py:131]: I think this debug message is redundant. 
> "Checking migration files for adjust messages"

Should be enough

File: saritasa_invocations/alembic.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    )
+    adjust_messages = config.get(
+        "alembic_adjust_messages",
+        (
+            "# ### commands auto generated by Alembic - please adjust! ###",
+            "# ### end Alembic commands ###",
+        ),
+    )
+    files_to_clean = collections.defaultdict(set)
+    for filename in os.listdir(migrations_folder):
```
Comment (ron8mcr) [inline saritasa_invocations/alembic.py:127]: Let's use `pathlib` here and below (start with https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob)

File: saritasa_invocations/alembic.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                files_to_clean[file_path].add(adjust_message)
+
+    if files_to_clean:
+        log_files_msg = "\n".join(
+            f"{file_name}: {';'.join(messages)}"
+            for file_name, messages in files_to_clean.items()
+        )
+        printing.print_error(
+            f"Adjust messages found in this migration files:\n{log_files_msg}",
+        )
```
Comment (ron8mcr) [inline saritasa_invocations/alembic.py:146]:
```suggestion
        log_files_msg = "\n\t".join(files_to_clean.keys())
        log_messages = "\n".join(adjust_messages)
        saritasa_invocations.print_error(
            f"Adjust messages found in this migration files:\n"
            f"\t{log_files_msg}\n"
            "Ensure that this files does not contain following:\n"
            f"{log_messages}",
        )
```

**Review by ron8mcr — CHANGES_REQUESTED** [review]

**Review by ron8mcr — APPROVED** [review]

**Review by NikAzanov — APPROVED** [review]

**Pull request merged**